src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./css/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        class="card__img" 
+        src="./images/imac.jpeg" 
+        alt="APPLE A1419"
+      >
+
+      <h3 class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__review">
+        <ul class="card__stars-list">
+          <li class="card__star"></li>
+          <li class="card__star"></li>
+          <li class="card__star"></li>
+          <li class="card__star"></li>
+          <li class="card__star"></li>
+        </ul>
+
+        <p class="card__number-of-reviews">Reviews: 5 </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-name">Price:</p>
+
+        <p class="card__price-value">$2,199</p>
+      </div>
+
+      <button class="card__buy-botton">
```

Comment:
Just spelling mistake in element name. Consider fixing.
```suggestion
      <button class="card__buy-button">
```